pkg/resource/v1/certconfig/resource.go

```diff
@@ -11,40 +13,90 @@ import (
 const (
 	// Name is the identifier of the resource.
 	Name = "certconfigv1"
+
+	// Suggested maximum number of CertConfigs returned in one List() call to
+	// K8s API. Server may choose to not support this so this is not strict
+	// maximum.
+	listCertConfigLimit = 25
 )
 
+type certificate struct {
+	name              certs.Cert
+	certConfigFactory func() (*v1alpha1.CertConfig, error)
+}
+
+// This is a list of certificates managed by this resource.
+var managedCertificates = []certificate{
```

Comment:
`certConfigFactory` methods are filled in next PR when `GetDesiredState()` is implemented.

Reply:
Same here with godoc.